tests/GuardTest.php

```diff
@@ -542,7 +542,7 @@ protected function getPackageProviders($app)
         return [SanctumServiceProvider::class];
     }
 
-    public function invalidTokenDataProvider(): array
+    public static function invalidTokenDataProvider(): array
```

Comment:
This fixes a phpunit warning, but I'm happy to separate it from the PR if it is too off topic for the PR.